cf_xarray/accessor.py

```diff
@@ -850,6 +850,16 @@ def describe(self):
         print(text)
 
     def get_valid_keys(self) -> Set[str]:
+
+        warnings.warn(
+            """Now called `keys` and `get_valid_keys` will
+                      be removed in a future version.""",
```

Comment:
```suggestion
                      be removed in a future version.",
```

Reply:
Same, changed separately and now on one line.